tests/benchmarks/entity_count.yml

```diff
@@ -1,10 +1,10 @@
-name: "ENTITY COUNT"
+name: "ENTITY_COUNT"
 remote:
   - setup: redisgraph-r5
   - type: oss-standalone
 dbconfig:
   - init_commands:
-    - "GRAPH.QUERY g UNWIND range(0, 5000000) AS x CREATE (:N)-[:R]->(:N)"
+    - '"GRAPH.QUERY" "g" "UNWIND range(0, 5000000) AS x CREATE (:N)-[:R]->(:N)"'
```

Comment:
was this change necessary?

Reply:
Yes, given we have spaces in the same argument ( and to make the tool agnostic ) we need the `"`